README.md

```diff
@@ -13,23 +13,48 @@ Basic:
 
 ```yaml
 steps:
-- uses: actions/checkout@master
+- uses: actions/checkout@v1.0.0
```

Comment:
No, we should not suggest this as the mainline (README).  the mainline is @v1 and binding to a specific patch release is a way to mitigate an issue and bind back if "latest v1" breaks you.  Actions `v1` is equal to "1.x".

Reply:
I addressed this feedback